### PR TITLE
[zh-cn] add translation for HighlightRegistry

### PR DESCRIPTION
**Pull request by ldt1997**

### CODE REVIEW

File: files/zh-cn/web/api/highlightregistry/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+## 实例属性
+
+_`HighlightRegistry` 接口不继承任何属性。_
+
+- {{domxref("HighlightRegistry.size")}} {{ReadOnlyInline}}
+  - : 返回当前已注册的 `Highlight` 对象的数量。
+
+## 实例方法
+
+_`HighlightRegistry` 接口不继承任何方法。_.
```
Comment (yin1999) [inline files/zh-cn/web/api/highlightregistry/index.md:25]:
```suggestion
_`HighlightRegistry` 接口不继承任何方法。_
```

File: files/zh-cn/web/api/highlightregistry/index.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+- {{domxref("HighlightRegistry.clear()")}}
+  - : 从注册表中移除所有 `Highlight` 对象。
+- {{domxref("HighlightRegistry.delete()")}}
+  - : 从注册表中删除指定的 `Highlight` 对象。
```
Comment (yin1999) [inline files/zh-cn/web/api/highlightregistry/index.md:30]:
```suggestion
  - : 从注册表中删除具有指定名称的 `Highlight` 对象。
```

File: files/zh-cn/web/api/highlightregistry/index.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+- {{domxref("HighlightRegistry.entries()")}}
+  - : 返回一个新的迭代器对象，该对象包含了注册表中的所有 `Highlight` 对象，按插入顺序排列。
+- {{domxref("HighlightRegistry.forEach()")}}
+  - : 按插入顺序为注册表中的每个 `Highlight` 对象执行一次提供的函数。
+- {{domxref("HighlightRegistry.get()")}}
+  - : 从注册表中获取指定的 `Highlight` 对象。
+- {{domxref("HighlightRegistry.has()")}}
+  - : 返回一个布尔值来指示对应的 `Highlight` 对象是否存在于该注册表中。
```
Comment (yin1999) [inline files/zh-cn/web/api/highlightregistry/index.md:38]:
```suggestion
  - : 返回一个指示对应的 `Highlight` 对象是否存在于该注册表中的布尔值。
```

File: files/zh-cn/web/api/highlightregistry/index.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+## 示例
+
+### 注册高亮
+
+以下示例演示了如何创建多个文本范围，然后为它们创建 `Highlight` 对象，并在 `HighlightRegistry` 中注册该高亮显示：
+
+#### HTML
+
+```html
+<p id="foo">CSS Custom Highlight API</p>
```
Comment (yin1999) [inline files/zh-cn/web/api/highlightregistry/index.md:55]: localize:

```suggestion
<p id="foo">CSS 自定义高亮 API</p>
```

File: files/zh-cn/web/api/highlightregistry/index.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+// 创建多个文本范围。
+const range1 = new Range();
+range1.setStart(text, 0);
+range1.setEnd(text, 3);
+
+const range2 = new Range();
+range2.setStart(text, 21);
+range2.setEnd(text, 24);
```
Comment (yin1999) [inline files/zh-cn/web/api/highlightregistry/index.md:82]:
```suggestion
range1.setStart(text, 0);
range1.setEnd(text, 3);

const range2 = new Range();
range2.setStart(text, 10);
range2.setEnd(text, 13);
```

**Review by yin1999 — COMMENTED** [review]
nits:

**Review by jasonren0403 — APPROVED** [review]
LGTM, Thanks @ldt1997, @skyclouds2001, and @yin1999

**Pull request merged**